Userland/Applications/Spreadsheet/JSIntegration.cpp

```diff
@@ -254,6 +254,17 @@ JS_DEFINE_NATIVE_FUNCTION(SheetGlobalObject::set_real_cell_contents)
 
 JS_DEFINE_NATIVE_FUNCTION(SheetGlobalObject::parse_cell_name)
 {
+    auto* this_object = vm.this_value(global_object).to_object(global_object);
+    if (!this_object)
+        return JS::js_null();
+
+    if (StringView("SheetGlobalObject") != this_object->class_name()) {
```

Comment:
```cpp
if (!is<SheetGlobalObject>(*this_object))
```

Reply:
Oh yeah, this file needs some _modernisation_